dags/pytorch_xla/configs/pytorchxla_torchbench_config.py

```diff
@@ -77,7 +77,7 @@ class R2_4(enum.Enum):
     TORCH = "torch==2.4.0"
     TORCHVISION = "torchvision==0.19.0"
     TORCHAUDIO = "torchaudio==2.4.0"
-    TORCH_XLA_GPU_DOCKER = "us-central1-docker.pkg.dev/tpu-pytorch-releases/docker/xla:r2.4.0-rc2_3.10_cuda_12.1"
+    TORCH_XLA_GPU_DOCKER = "us-central1-docker.pkg.dev/tpu-pytorch-releases/docker/xla:r2.4.0-rc8_3.10_cuda_12.1"
```

Comment:
Do you want to use the RC or the final release tag?

Reply:
Updated